pkg/registry/common/memory/ns_memory_server.go

```diff
@@ -0,0 +1,62 @@
+// Copyright (c) 2020 Doc.ai and/or its affiliates.
```

Comment:
I think it should be under common/registry/memory? Thoughts?

Reply:
Thanks, fixed.